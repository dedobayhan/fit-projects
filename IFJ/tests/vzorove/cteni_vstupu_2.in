"\"Po ponurém poli poletovali pakoni, \nvazne\"\n"
"5 x \$4 = \$20\n"
"prvni$druha$treti$ctvrta$serazena$mucha"

