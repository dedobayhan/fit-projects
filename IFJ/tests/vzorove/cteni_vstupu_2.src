var string s1, string s2, string s3, string nl;

void main()
{
  nl := "\n";

  s1 := readstring();
  printstring(s1);

  s2 := readstring();
  printstring(s2);

  s3 := readstring();
  printstring(s3);
  printstring(nl);
  printstring(sort(s3));
  
  printstring(nl);
  printstring(nl);
  printint(strlen(s1));
  printstring(nl);
  printint(strlen(s2));
  printstring(nl);
  printint(strlen(s3));
  printstring(nl);
}
