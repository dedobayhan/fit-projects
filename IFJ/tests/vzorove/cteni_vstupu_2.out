"Po ponurém poli poletovali pakoni, 
vazne"
5 x $4 = $20
prvnidruhatretictvrtaserazenamucha
ctvrtadruhamuchaprvniserazenatreti

44
13
34
